packages/core-utils/src/deprecated.js

```diff
@@ -316,3 +316,17 @@ export function summarizeQuery(query, locations = []) {
     : require("./itinerary").toSentenceCase(query.mode);
   return `${mode} from ${from} to ${to}`;
 }
+
+export function getTimeZoneOffset(itinerary) {
```

Comment:
We should probably add a deprecation warning

```suggestion
export function getTimeZoneOffset(itinerary) {
logDeprecationWarning("getTimeZoneOffset");
```

Reply:
Yes, good one, thank you (https://github.com/opentripplanner/otp-ui/pull/398/commits/57b99e9ca4c62ccf622b5d38a14be4d6b6754fc6).